agent_token_test.go

```diff
@@ -0,0 +1,111 @@
+package tfe
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAgentTokensList(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	apTest, apTestCleanup := createAgentPool(t, client, nil)
+	defer apTestCleanup()
+
+	agentToken1, agentToken1Cleanup := createAgentToken(t, client, apTest)
+	defer agentToken1Cleanup()
+	agentToken2, agentToken2Cleanup := createAgentToken(t, client, apTest)
+	defer agentToken2Cleanup()
+
+	t.Run("with no list options", func(t *testing.T) {
+		tokenlist, err := client.AgentTokens.List(ctx, apTest.ID)
+		require.NoError(t, err)
+		assert.Contains(t, tokenlist.Items, agentToken1)
+		assert.Contains(t, tokenlist.Items, agentToken2)
+
+		t.Skip("paging not supported yet in API")
```

Comment:
Pagination is supported in this API; this should be removed.

Reply:
👍